### PR TITLE
feature: edtr-io integration

### PR DESCRIPTION
**Pull request by Knorrke**

### CODE REVIEW

File: packages/private/edtr-io/public/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
Comment (inyono) [inline packages/private/edtr-io/public/index.html]: no longer needed

File: packages/private/edtr-io/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (inyono) [inline packages/private/edtr-io/public/manifest.json]: no longer needed

File: packages/private/edtr-io/src/editor.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  video: videoPlugin
+}
+
+export function Editor(props : React.PropsWithChildren<{initialState: { plugin: string, state: unknown }, editable: boolean}>) {
+  return (
+    <Core plugins={plugins} defaultPlugin="text" initialState={props.initialState} editable={props.editable}>{props.children}</Core>
+  )
+}
+
+export function Controls() {
```
Comment (inyono) [inline packages/private/edtr-io/src/editor.tsx]: ?

File: packages/private/edtr-io/src/editor.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  layout: layoutPlugin,
+  rows: rowsPlugin,
+  scMcExercise: scMcExercisePlugin,
+  solution: solutionPlugin,
+  spoiler: spoilerPlugin,
+  text: textPlugin,
+  video: videoPlugin
+}
+
+export function Editor(props : React.PropsWithChildren<{initialState: { plugin: string, state: unknown }, editable: boolean}>) {
```
Comment (inyono) [inline packages/private/edtr-io/src/editor.tsx]: EditorProps["initialState"] resp. EditorProps["editable"]

File: packages/private/edtr-io/src/hooks.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { selectors, useStore } from '@edtr-io/core'
```
Comment (inyono) [inline packages/private/edtr-io/src/hooks.ts]: Probably no longer needed after integrating it completely?

Comment (metzgaria) [reply packages/private/edtr-io/src/hooks.ts]: Is it ok to keep it till edtr.io is integrated completely?

Comment (inyono) [reply packages/private/edtr-io/src/hooks.ts]: yep

**Comment by inyono** [comment]
`yarn upgrade` and committing the `yarn.lock` file should fix the build

**Pull request merged**